src/vue/modules/issuance-explorer/components/issuances-table.vue

```diff
@@ -80,3 +124,38 @@ export default {
   },
 }
 </script>
+
+<style lang="scss">
+  @import "~@scss/variables";
+ .issuance_table {
+   .request-state {
```

Comment:
`issuance-table__request-state`
Every component is a BEM block. In this case, `request-state` isn't a new stand-alone block, so styles of this DOM-elements refers to the `issuance-table` block.